cypress/e2e/spec_main.cy.js

```diff
@@ -0,0 +1,39 @@
+import { HomePage } from "../support/components/HomePage";
+import data from "../fixtures/data.json";
+
+describe('main page tests', () => {
+  const homePage = new HomePage();
+
+  beforeEach('opens main page', () => {
+    cy.visit('/');
+  })
+
+  it('resets the input', () => {
+    homePage.typeInputQuery(data.query);
+    homePage.clickInputResetButton();
+
+    homePage.getInput().should('have.value', '');
+  })
+
+  it('shows message on invalid login credentials', () => {
```

Comment:
I get an issue with the callback upon submitting the form with Cypress (the error can be visible in the devtools console), thus I don't get the popup message that I need. There is a redirection on the POST-request, so this behaviour might be connected to the following [Cypress issue](https://github.com/cypress-io/cypress/issues/24700). However, I decided to use intercept to get the script for the needed popup and assert its content. The callback issue doesn't seem to be connected with the fixtures, nor with the clicking the submit button. Would you have any ideas on how to make this work?